viewing_party/party.py

```diff
@@ -1,23 +1,128 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title == None or genre == None or rating == None:
+        new_movie = None
+        return new_movie
+    else:
+        new_movie = {'title':movie_title,'genre':genre, 'rating':rating}
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    count = 0
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            add_to_watched(user_data,movie)
+            del(user_data["watchlist"][count])
+        count += 1
+    return user_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
+def get_watched_avg_rating(user_data):
+    sum = 0
+    for i in range (0,len(user_data["watched"])):
+        print(user_data["watched"][i]["rating"])

```

Comment:
remove print statements after debugging/testing. Print statements should only be added if it related to the user experience